### PR TITLE
Make key environment as a mapping type rather than a sequence type.

### PR DESCRIPTION
**Pull request by rupakg**
[Finishes #95666094]

### CODE REVIEW

**Comment by dharmamike** [comment]
If I import a docker-compose.yml with a sequence type, Kwalify will mark the key line as having an error.  Is that really what we want?

File: schema.yml [hunk 1]
```diff
@@ -80,9 +80,7 @@ mapping:
 
           Environment variables with only a key are resolved to their values on the machine Compose is running on, which
```
Comment (dharmamike) [inline schema.yml]: The description here says "You can use an array"  but that's only partially true now.

**Comment by dharmamike** [comment]
:+1: with an updated description

File: schema.yml (hunk 1 above)
Comment (rupakg) [inline schema.yml]: good catch, missed that. I will update it to say "You can use an array or a hash."

**Pull request merged**